nncf/tensorflow/graph/patterns.py

```diff
@@ -49,7 +49,7 @@
 
 POOLING = N('AveragePooling2D') | N('AveragePooling3D') | N('GlobalAveragePooling2D') | N('GlobalAveragePooling3D')
 
-SINGLE_OPS = POOLING | N('Average') | N('LayerNormalization')
+SINGLE_OPS = POOLING | N('Average') | N('LayerNormalization') | N('UpSampling2D')
```

Comment:
@ l-bat Could you include this change in your PR? Because the models need to be retrained, and you will do it all at once.